config.dev.yaml

```diff
@@ -75,6 +75,14 @@ services:
         query: '{{ default(request.query, {}) }}'
         headers: '{{request.headers}}'
         body: '{{request.body}}'
+      wdqsapi_req:
+        method: 'post'
+        uri: 'https://query.wikidata.org/bigdata/namespace/wdq/sparql'
+        headers:
```

Comment:
Should also probably add `content-type: application/json` to the list of headers